packages/test-base/src/commonTest/kotlin/io/realm/kotlin/test/common/notifications/SystemNotificationTests.kt

```diff
@@ -76,5 +76,9 @@ class SystemNotificationTests {
             writer1.write { copyToRealm(Sample()) }
             writer2.write { copyToRealm(Sample()) }
         }
+        writer1.close()
+        writer2.close()
+        liveRealmContext.close()
```

Comment:
Is this actually green on CI? I tried to clean this up as part of another PR but ended up splitting into this separate issue #1563

Reply:
Hmm. Maybe closing the writers individually first make it work.

Reply:
Yes, this is working on GHA.